jstests/mongo_rocks/stats_tickers.js

```diff
@@ -8,8 +8,8 @@
     if (status.storageEngine.name == 'rocksdb') {
```

Comment:
Let's please make this script work for any engine specified by explicitly setting rocksdb when starting Mongod.
```
var conn = MongoRunner.runMongod({
  storageEngine: 'rocksdb',
});
```